test/gtest/uct/test_md.cc

```diff
@@ -820,20 +821,33 @@ UCS_TEST_SKIP_COND_P(test_md, invalidate, !check_caps(UCT_MD_FLAG_INVALIDATE))
         ASSERT_UCS_OK(status);
 
         bool is_unique = keys_set.insert(key).second;
-        ASSERT_TRUE(is_unique) << keys_set.size()
-                               << "-th key is not unique";
+        if (!is_unique) {
+            if (keys_set.size() < 64) {
```

Comment:
why 64?

Reply:
why we can tolerate non-unique keys now?

Reply:
@evgeny-leksikov can you please take a look?